src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,9 +1,13 @@
+UPDATE tab_calculation_method_rates SET formula='f_onto * f_disease * C_0 * V' WHERE calculation_method='CompoundCommon' AND formula_rate='PARAM_StartAmount_Protein';
+INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('DiseaseStates','One_Rate','1','Fraction');

```

Comment:
I think the dimension of the "disease factor" and its formula should be `dimensionless`, not `fraction`.
Especially the factor could become also >1

Reply:
yep, Ontogeny factor should be dimensionless as well...